site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -66,6 +66,49 @@ To install `gcloud` and `kubectl`, perform the following steps:
 
 [gcloud-install]: https://cloud.google.com/sdk/docs/quickstarts
 
+### Choosing a GKE cluster mode
+
+A [cluster][cluster] consists of at least one *control plane* machine and multiple worker machines called *nodes*. In Google Kubernetes Engine, nodes are [Compute Engine virtual machine][vms] instances that run the Kubernetes processes necessary to make them part of the cluster.
+
+Agones supports GKE Standard mode and has alpha support for GKE Autopilot mode.
+
+#### Agones on GKE Autopilot
+
+Autopilot is GKE's fully-managed mode. GKE configures, maintains, scales, and
+upgrades nodes for you, which can reduce your maintenance and operating
+overhead. You only pay for the resources requested by your running Pods, and
+you don't pay for unused node capacity or Kubernetes system workloads.
+
+This section describes the Agones-specific considerations in Autopilot
+clusters. For a general comparison between Autopilot and Standard, refer to
+[Choose a GKE mode of operation](https://cloud.google.com/kubernetes-engine/docs/concepts/choose-cluster-mode).
+
+Autopilot nodes are, by default, optimized for most workloads. If you have
+specific hardware needs such as Arm architecture, specific CPU platforms, or
+hyperthreading disabled, you can specify
+[compute classes](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-compute-classes)
+for those workloads.
+
+Autopilot has pre-configured opinionated constraints. You might want to use GKE
+Standard mode if these constraints aren't right for you:
+
+*  **Operating system:** No Windows containers.
+*  **Resource requests:** Pre-determined
+   [minimum Pod resource requests](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-resource-requests#min-max-requests).
+   If your game servers require less than those minimums, use GKE Standard.
+*  **[Scheduling strategy](https://pkg.go.dev/agones.dev/agones/pkg/apis#SchedulingStrategy):** `Packed` is supported, which is the Agones default.
+   `Distributed` is not supported.
+*  **[Host port policy]({{<ref "/docs/reference/agones_crd_api_reference#agones.dev/v1.GameServerPort">}}):** `Dynamic` is supported, which is the Agones default.
+   `Static` and `Passthrough` are not supported.
+*  **Seccomp profile:** Agones sets the seccomp profile to `Unconfined` to
+   avoid unexpected container creation delays that might occur because
+   Autopilot enables the `RuntimeDefault` seccomp profile.
+*  **Pod disruption:** `eviction.safe: Never` is supported, which is the Agones
```

Comment:
Suggestion: link to the eviction docs, because I can't remember which mode is which (or I may not know) 😃